.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "5.0"
```

Comment:
I would actually just do one, maybe two versions of node.  Otherwise we might hit Valve's rate limit?